image/Dockerfile.engine-dm

```diff
@@ -37,19 +37,22 @@ ARG BUILDTIME
 ARG PLATFORM
 ARG PRODUCT
 ARG DEFAULT_PRODUCT_LICENSE
+ARG TINI_COMMIT
 ENV VERSION ${VERSION}
 ENV GITCOMMIT ${GITCOMMIT}
 ENV BUILDTIME ${BUILDTIME}
 ENV PLATFORM ${PLATFORM}
 ENV PRODUCT ${PRODUCT}
 ENV DEFAULT_PRODUCT_LICENSE ${DEFAULT_PRODUCT_LICENSE}
+ENV TINI_COMMIT ${TINI_COMMIT}
+SHELL ["/bin/bash", "-c"]
 # TODO The way we set the version could easily be simplified not to depend on hack/...
-RUN bash ./hack/make/.go-autogen
-RUN go build -o /dockerd \
-    -tags 'autogen apparmor seccomp selinux journald' \
+RUN source ./hack/make/.go-autogen; \
+    go build -o /sbin/dockerd \
```

Comment:
This `/sbin/dockerd` will cause a problem in the last stage. There's a `COPY --from` in line 80 that uses `/dockerd` as source.

This lead to a next question, why are there two relatively similar `Dockerfile.engine*` files that we have to maintain?

At least we should use the same pathes in both Dockerfiles, so the COPY instruction must be updated.

Reply:
🤦‍♂ I worked on one file, then copied to the other file, but forgot they had different paths.